app/controllers/projects_controller.rb

```diff
@@ -343,6 +343,6 @@ def return_context
     render_404 && return if @project.blank?
     return unless @project.deleted?
     flash[:alert] = 'The project you requested had been deleted.'
-    redirect_to user_path(@user)
+    check_redirect "/#{@user.username}"
```

Comment:
yeah it wouldn't...  I can't see a warning for this line you changed... If it's there and I didn't see it please point me to it... This part is all ready to be replaced... I have already defined the routes of the projects. Issues not yet merged but they're also ready.